projects/storefrontlib/src/cms-components/configurator/commons/config-attribute-types/config-attribute-drop-down/config-attribute-drop-down.component.html

```diff
@@ -0,0 +1,23 @@
+<div id="{{ uiKeyGenerator.createAttributeIdForConfigurator(attribute) }}">
+  <div class="row">
+    <div class="col-md-9 col-lg-9">
+      <div class="form-group">
+        <ng-container *ngIf="attribute.values">
```

Comment:
Why is there a ng-container used?
The *ngIf could be added to the div below.
"attribute.values && attribute.values.length !== 0"

Reply:
I used the address-form.component.html as template for the drop-down. There the ng-container is also used.

Reply:
done